website/members/management/commands/createmember.py

```diff
@@ -0,0 +1,190 @@
+                    except exceptions.ValidationError as err:
+                        self.stderr.write("\n".join(err.messages))
+                        response = input(
+                            "Bypass password validation and create user anyway? [y/N]: "
+                        )
+                        if response.lower() != "y":
+                            continue
+                    user_data[PASSWORD_FIELD] = password
+            else:
+                # Non-interactive mode.
+                # Use password from environment variable, if provided.
+                if (
+                    PASSWORD_FIELD in user_data
+                    and "DJANGO_SUPERUSER_PASSWORD" in os.environ
+                ):
+                    user_data[PASSWORD_FIELD] = os.environ["DJANGO_SUPERUSER_PASSWORD"]
+                # Use username from environment variable, if not provided in
+                # options.
+                if username is None:
+                    username = os.environ.get(
+                        "DJANGO_SUPERUSER_" + self.UserModel.USERNAME_FIELD.upper()
+                    )
+                if username is None:
+                    raise CommandError(
+                        "You must use --%s with --noinput."
+                        % self.UserModel.USERNAME_FIELD
+                    )
+                else:
+                    error_msg = self._validate_username(
+                        username, verbose_field_name, database
+                    )
+                    if error_msg:
+                        raise CommandError(error_msg)
+
+                user_data[self.UserModel.USERNAME_FIELD] = username
+                for field_name in self.UserModel.REQUIRED_FIELDS:
+                    env_var = "DJANGO_SUPERUSER_" + field_name.upper()
+                    value = options[field_name] or os.environ.get(env_var)
+                    if not value:
+                        raise CommandError(
+                            "You must use --%s with --noinput." % field_name
+                        )
+                    field = self.UserModel._meta.get_field(field_name)
+                    user_data[field_name] = field.clean(value, None)
+                    if field.many_to_many and isinstance(user_data[field_name], str):
+                        user_data[field_name] = [
+                            pk.strip() for pk in user_data[field_name].split(",")
+                        ]
+
+            user = self.UserModel._default_manager.db_manager(
+                database
+            ).create_superuser(**user_data)
+            # Changed from django/contrib/auth/management/commands/createsuperuser.py
+            Profile.objects.create(user=user)
+            Membership.objects.create(
+                user=user, type=Membership.MEMBER, since=timezone.now()
+            )
+
+            if options["verbosity"] >= 1:
+                self.stdout.write("Superuser created successfully.")
```

Comment:
If both createsuperuser and createmember are available, I think the wording should be consistent about this being about members, not just superusers